alpha/fsm.h

```diff
@@ -0,0 +1,55 @@
+#ifndef __TS_FSM__
+#define __TS_FSM__
+
+#include "src\FunctionFsm\src\FunctionFSM.h"
+
+enum TS_PowerState
+{
+  LOW_POWER,
+  HIGH_POWER,
+};
+
+class _TS_FSM
+{
+  public:
```

Comment:
hmm, is there an accessor for another task to check what power state this is in ?
main radio task needs to check to switch between modes e.g. wifi or not